### PR TITLE
Person profile page (Part 2/2): Logic and API call

### PR DESCRIPTION
**Pull request by lin8231**
* Person profile page pretty much completed
* Things are rendered dynamically now
* Routing params working
* UI-wise nothing changed. Biggest observable difference is that now routing param is fully functional 
* Added a few tests
* Needs some updating to match updated /person/:id API from the backend (will make new issue for this), and popups and routing logic - will be added in the next PR so that this PR doesn't get too big
* more tests to come with ^ this issue

## Demo
See URL with routing param, data is pulled from backend using that id:
image

### CODE REVIEW

**Review by lusicle — APPROVED** [review]
lgtm 😄

File: forgettable-frontend/src/pages/PersonPage/PersonPage.js [hunk 1]
```diff
@@ -1,30 +1,106 @@
...
+      </div>
+    );
+  };
+
   return (
     <div className={classes.PersonPage}>
       <PersonDrawer
         open={true}
-        img="https://user-images.githubusercontent.com/62003343/158300159-495244dd-f05e-4384-9fac-ddf88a06be39.gif"
+        img={person.img}
```
Comment (noahjade) [inline forgettable-frontend/src/pages/PersonPage/PersonPage.js:86]: have you changed anything yet to make it support the image buffer that gets returned by the api? just asking cause im about to try figure that out haha

**Review by noahjade — APPROVED** [review]
Amazing work! :tada: I left a question about the image buffer but thats more for me lmao so non blocking :)

File: forgettable-frontend/src/pages/PersonPage/PersonPage.js (hunk 1 above)
Comment (lin8231) [reply forgettable-frontend/src/pages/PersonPage/PersonPage.js:86]: Oooh I have not, didn't occur to me that that doesn't work haha I just thought the person didn't have a photo. Will include this in the PR to update the API call :D

**Pull request merged**